uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request-ctrl.js

```diff
@@ -45,6 +45,11 @@ define([
        * @return {Promise}
        */
       vm.calculateToilExpiryDate = function () {
+        //do not calculate expiry date if its already set by manager for this leave request
+        if (vm.isRole('owner') && vm.isMode('edit')) {
+          return $q.resolve(vm.request.toil_expiry_date);
```

Comment:
Three things

1. You are referring to the manager in this comment (`if its already set by manager for this leave request`) but in the code you are referring the owner?
2. If you need to skip the calculation, then i think it's better to not just call the method in the first place, instead of having some exception handling in it
3. There is a TODO for this method, is it still valid?